src/main/java/com/parasoft/parabank/web/form/AdminForm.java

```diff
@@ -114,6 +115,8 @@ public void setParameters(final Map<String, String> parameters) {
         loanProcessor = parameters.get(AdminParameters.LOAN_PROCESSOR);
         varString=parameters.get(AdminParameters.LOAN_PROCESSOR_THRESHOLD);
         loanProcessorThreshold = Integer.parseInt(varString == null || varString.isEmpty() ? "20" : varString);
+        varString = parameters.get(AdminParameters.WEB_AUTHENTICATION_ENABLED);
+        webAuthenticationEnabled = Boolean.parseBoolean(varString == null || varString.isEmpty() ? "false" : "true");
```

Comment:
This doesn't look right.  If varString is "false" it will instead use "true" because varString is not null and not empty.